fwb/app/(auth)/(routes)/detail/page.tsx

```diff
@@ -1,15 +1,73 @@
+import { FilterProvider } from '@/components/ui/explore/filter_context'
 import { auth } from '@clerk/nextjs'
-import DetailPage from './detail'
 import { redirect } from 'next/navigation'
-import {
-  DetailData,
-  CompanyAndDiscounts,
-  DiscountDataDetail,
-} from '@/app/types/types'
-import {
-  FilterContext,
-  FilterProvider,
-} from '@/components/ui/explore/filter_context'
+import DetailPage from './detail'
+
+export async function getCompanyId(companyName: string) {
```

Comment:
This is causing the build problem, remove any export functions into a separate util file, and import it from there. 
So in this case, shift out getCompanyId() & getCompanyDiscountsTable(). handleSearch can stay since we're not exporting it.

Pages don't allow for us to have multiple exports like this